### DISCUSSION TITLE
YOLO

### DISCUSSION THREAD

**Discussion by Belikeoscar** [community/community#18542] [Other Feature Feedback, Questions, & Ideas] [answered]
I feel having the yolo badge does not pass the right message especially to recruiters. Reasons for having one could vary, like having to work on friendly projects that wouldn't require reviews.

**Comment by Zerotask** [comment, marked as answer]
Recruiters won't care if you have that badge or not. Besides that you have also the option to hide badges

**Reply by Belikeoscar** [reply]
Alright

**Reply by rahmatsubandi** [reply]
> Recruiters won't care if you have that badge or not. Besides that you have also the option to hide badges

Hi, where can I get YOLO?

**Reply by Belikeoscar** [reply]
You get the badge if you merge code from a pull request without conducting code reviews

**Reply by floer32** [reply]
I would show my badges if this badge wasn't one of them. It would be preferable if this "achievement" was removed, because some of the other achievements communicate useful information.

**Reply by MichaelWheldon** [reply]
I can hide this trash? Can you please let me know how to do that 🙏

**Reply by danielepiccone** [reply]
I don't understand why this is even an achievement, it has a negative connotation, and specially irrelevant for private projects.

**Reply by DevCleverton** [reply]
<img style="display: block;-webkit-user-select: none;margin: auto;background-color: hsl(0, 0%, 90%);transition: background-color 300ms;" src="https://github.githubassets.com/images/modules/profile/achievements/yolo-default.png">

**Reply by fritzmg** [reply]
> Besides that you have also the option to hide badges

But that would hide all the badges. This discussion is about the YOLO badge and its negative connotation.

**Reply by strom2357** [reply]
This response -- "recruiters won't care" -- felt really dismissive.  Why not acknowledge OP's valid and shared concern about sending the right message on a professional profile?

**Reply by 7iliama1** [reply]
👍🏼

**Comment by sampi** [comment]
sometimes you merge something but you still want the "paper trail" or perhaps it will be reviewed asynchronously the next morning.

**Reply by Belikeoscar** [reply]
that too, I just hope it doesn't become a basis of judgement by people.

**Reply by Pavel8913** [reply]
Alright

**Comment by floer32** [comment]
This badge _especially_ shouldn't show up when it's about a private repository, or your own repository with no other contributors. But especially private repositories; it doesn't even make sense in that case.

**Comment by MichaelWheldon** [comment]
Oh hey, another completely stupid, bloated and distracting social feature on github. Just because badges and the gamification of the internet didn’t pan out a decade ago, why not try again?!

**Reply by ThomasThelen** [reply]
Did you forget that GitHub has been a social coding platform from the beginning?

**Reply by stratigos** [reply]
We havent forgotten that GitHub used to be logical and reasonable.

**Comment by sthagen** [comment]
The criteria of that badge seem useless, as peer reviews not executed by peers in an agreed time window often are a valid process implementation. So, this badge takes all other badges down with it (as I understand I can only decide for all to show or not, right)?

(Always wanted to end the text of an "answer" on a social media site with a question mark ...) 

❔

**Reply by Belikeoscar** [reply]
We can honestly do without the Yolo, don't you think so? 
(😏 ? )

**Comment by fritzmg** [comment]
I agree that this badge should be removed, as it conveys a negative connotation about something that can be completely valid and normal.

**Comment by VictorHugoAguilar** [comment]
> 

It depends on the recruiter, but the difficult thing is to get it

**Reply by sthagen** [reply]
No. That wun is easy to get on GitHub.

**Comment by Kyle772** [comment]
I agree that it's not a very good badge. Why would I want other developers to think I merge without reviews?

**Reply by ThomasThelen** [reply]
GitHub is a social programming platform - you don't have to use it specifically for work or serious projects where there are strict standards in place. At the end of the day, judging a developer by gamification badges is probably not the best way to assess skill or quality.

**Reply by 7iliama1** [reply]
agree 👍🏼

**Comment by sthagen** [comment]
Also, many pull requests I participate in are not at all about software development but instead say represent drafts of meeting minutes, or examples for specifications, ...

**Comment by nragland37** [comment]
If you're having doubts and don't want to hide all your badges. Simply, click on the specific badge in your profile that you'd like to hide. 

https://user-images.githubusercontent.com/92565109/194934684-f0b669ae-0c9c-4c82-8a80-fac0b351f971.mp4

**Reply by stratigos** [reply]
Thank you for showing this - this comment was buried for me when I first viewed the page, I had to expand comments to see it.

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide

**Reply by julienheinen** [reply]
ohhhhhh it's beautiful

**Comment by KeplerDE** [comment]
Super

**Comment by julienheinen** [comment]
I've been trying to get it for about 1 hour. It's much more annoying than trying to fix an unsolvable error in a program.

**Comment by stratigos** [comment]
I agree that the `YOLO` badge is not at all something positive. It represents an antipattern. I in no way shape or form want to be associated with the `YOLO` badge or what it means. Sometimes I have repositories for which I am the only maintainer, and therefore, I am the only one who can merge a PR. This is not a "YOLO" event, its intentional in that case, and in no way shape or form represents my values or how I would work on a team. I strongly advocate for peer review, and take peer review process very seriously. 

I like my other badges, I want `YOLO` removed from my profile. Essentially, I want to be able to pick and choose which badges display, given that this and potentially others are antipatterns and not bragworthy at all. 

If anything, remove `YOLO`, its a badge for the sake of having a badge, and does not represent any sort of positive action. 

The recruiter remark is irrelevant. I do not interact with recruiters. I am not at all interested in what a recruiter thinks of my GitHub profile. I would instead prefer to reflect my values as an engineer to other engineers with my GitHub profile.

**Reply by stratigos** [reply]
After expanding the comments  - which wasnt immediately clear to me when I first arrived on the page - I have learned how to hide a single, specific badge.

I still think its an illogical choice, and  this is a platform for engineers. Remove the YOLO badge entirely, its in no way shape or form a positive value!

**Comment by lahirusamishka** [comment]
The `YOLO` badge is an important indicator of a developer's behavior. While it may suggest practices like merging code without a proper review, it also offers insights into their habits. Allowing the badge to change dynamically could be beneficial.

For example, developers who engage in negative practices could receive "bad badges" as feedback. This would encourage them to prioritize best practices to improve their badges over time. As they address these areas, they could work toward earning "good badges" that reflect positive development behaviors.

**Comment by Christcool** [comment]


**Reply by julienheinen** [reply]
Salut, 
Ici il s'agit d'un badge qu'on peut obtenir sur github, sur ton profil.

En ce qui concerne ton soucis, avec Yolo que j'ai déjà utilisé (le hasard fait bien les choses), que se passe t'il concrètement ? 
J'imagine que tu utilises Yolo pour de la détection en temps réel, via une caméra par exemple ?
Et tu dois sûrement avoir un dataset pour ton réseau de neurones en CNN.
Si c'est bien le cas, quel dataset utilises tu ? En terme de ratio images/maladies à combien es tu ? Par exemple est ce que tu as 30 maladies différentes, pour environ 120 images pour chaque maladies. 

Tu peux me contacter en privée 😉

**Reply by Christcool** [reply]
comment puis-je vous contacter en privée?

**Comment by ninjacode911** [comment]
Hey @Christcool,

  Wrong thread — this discussion is about GitHub's YOLO achievement badge, not the YOLO object detection model.

  For help with YOLOv_ plant disease detection, try posting in:
  - https://github.com/ultralytics/ultralytics/discussions (if you're using YOLOv8/v11)
  - https://stackoverflow.com/questions/tagged/yolo with the yolo tag
  - r/computervision or r/MachineLearning on Reddit

  You'll get much better help there from people experienced with object detection tuning.
